nixos/modules/security/pam.nix

```diff
@@ -895,6 +895,61 @@ in
         runuser-l = { rootOK = true; unixAuth = false; };
       };
 
+    security.apparmor.includes."abstractions/pam" = let
+      isEnabled = test: fold or false (map test (attrValues config.security.pam.services));
+      in ''
+      ${lib.concatMapStringsSep "\n"
```

Comment:
We have to do something about readability again. I only see ${}${{}$} here and can't really figure out what might be right.

Reply:
I've rewritten using `''` and `+`. Concerning the rightness, this is allowing PAM to load the PAM modules enabled in the Nix configuration.